CMakeLists.txt

```diff
@@ -292,12 +292,14 @@ if(BUILD_EXAMPLES AND NOT WIN32)
     target_compile_definitions(hello_module PRIVATE ${qjs_defines})
     target_link_libraries(hello_module ${qjs_libs})
 
-    if(NOT WIN32)
+    # if(NOT WIN32)
         add_library(fib MODULE examples/fib.c)
         set_target_properties(fib PROPERTIES
             PREFIX ""
             C_VISIBILITY_PRESET default
         )
+        target_include_directories(fib PRIVATE .)
+        target_link_libraries(fib PRIVATE qjs)
```

Comment:
is there no way to do lazy symbol binding on windows?

Reply:
i am not sure